usage/bigquery.go

```diff
@@ -28,16 +29,16 @@ var allowedTimeSteps = map[string]bool{
 }
 
 type UsageSummaryRow struct {
-	UserID    string `bigquery:"user_id"`
-	CreatorID string `bigquery:"creator_id"`
-
+	UserID            string  `bigquery:"user_id"`
+	CreatorID         string  `bigquery:"creator_id"`
 	DeliveryUsageMins float64 `bigquery:"delivery_usage_mins"`
-	TotalUsageMins   float64 `bigquery:"transcode_total_usage_mins"`
-	StorageUsageMins float64 `bigquery:"storage_usage_mins"`
+	TotalUsageMins    float64 `bigquery:"transcode_total_usage_mins"`
+	StorageUsageMins  float64 `bigquery:"storage_usage_mins"`
 }
 
 type BigQuery interface {
 	QueryUsageSummary(ctx context.Context, userID string, creatorID string, spec QuerySpec) (*UsageSummaryRow, error)
+	QueryUsersUsage(ctx context.Context, userIDs []string, spec QuerySpec) (*[]UsageSummaryRow, error)
```

Comment:
**nit:** If you go with "all users" instead of "list of users", this can be renamed to `QueryAllUsersUsage` for clarity.